freezegun/api.py

```diff
@@ -170,7 +170,7 @@ def get_current_time():
     return freeze_factories[-1]()
 
 
-def fake_time():
+def fake_time(*args, **kwargs):
```

Comment:
Why *args, **kwargs here? That seems very wrong to me.

Reply:
Hello @boxed , 
Thanks for your review. May I humbly ask you to elaborate? 

I think I made an effort to explain the reproduction of the bug, both in the description and in the test.
If I'm off track, can you possibly guide me to a solution that you think is more appropriate?

Reply:
@JulianMaurin this is breaking the API typing on `api.pyi`, so users that have mypy will have problems. I also think you can add typed arguments to satisfy your needs with default `arguments=None` so it will also not break current applications and maintain the types well defined.

Reply:
I thought that this version does not break the application, and in addition  `fake_time` function is not defined in `api.pyi`, but maybe I did not understand something.

Would you prefer this version (with a docstring ofc):
```suggestion
def fake_time(_ : Any):
```